src/controllers/userController.js

```diff
@@ -111,6 +159,18 @@ const getCurrentUserWithPurchasedAlbums = async (req, res) => {
     user: userWithAlbums,
     purchasedAlbumCount: userWithAlbums.purchasedAlbums.length,
   });
+  /*
+     #swagger.summary = 'Fetch a user by id and all albums they purchased.'
+     #swagger.parameters['id'] = {
+        description: 'Mongo ObjectID of the user to fetch',
+        schema:  { $ref: '#/definitions/UserWithAlbums' }
+     }
```

Comment:
This endpoint doesn't take the id in the path/body/query paramaters, it just grabs the user id from the authenticated cookie. So we can remove these lines